src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs

```diff
@@ -43,7 +44,7 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
                     {Configuration.TimestampColumnName} BIGINT NOT NULL,
                     {Configuration.ManifestColumnName} VARCHAR(500) NOT NULL,
                     {Configuration.PayloadColumnName} {storedAs} NOT NULL,
-                    {Configuration.TagsColumnName} VARCHAR(100) NULL,
+                    {Configuration.TagsColumnName} VARCHAR({tagsColumnSize}) NULL,
```

Comment:
From a SQL standpoint, this should be fine - changing the size of this field is won't affect existing users' ability to query.